src/main/java/org/dswarm/graph/hash/HashUtils.java

```diff
@@ -16,12 +16,18 @@
  */
 package org.dswarm.graph.hash;
 
+import com.github.emboss.siphash.SipKey;
+
 /**
  *
  */
 public final class HashUtils {
 
-	private HashUtils() {}
+	public static final SipKey	SPEC_KEY	= new SipKey(HashUtils.bytesOf(0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0a, 0x0b,
+													0x0c, 0x0d, 0x0e, 0x0f));
+
```

Comment:
I think I just took them from the test in the original siphash java impl project, see https://github.com/emboss/siphash-java/blob/master/src/test/java/com/github/emboss/siphash/SipHashTest.java#L12
dunno, whether something about the bytes can be found at https://131002.net/siphash/